distribution/docs/src/main/resources/_contents/_running/starting-intro-contents.adoc

```diff
@@ -92,7 +92,7 @@ Because ${branding} is built on top of Apache Karaf, ${branding} can use the Kar
 .Within the ${branding} console
 ----
 ${branding-lowercase}${at-symbol}local> feature:install wrapper
-${branding-lowercase}${at-symbol}local> wrapper:install -s AUTO_START -n ${ddf-branding-lowercase} -d ${ddf-branding-lowercase} -D "${branding} Service"
+${branding-lowercase}${at-symbol}local> wrapper:install -s AUTO_START -n ${branding-lowercase} -d ${branding-lowercase} -D "${branding} Service"
```

Comment:
`INSTALLATION_HOME`, `<DDF_HOME>`, and `DDF_INSTALL` are all used in this file. I'm not sure what the convention is, but they all should probably be something like `<DDF_INSTALLATION_HOME>`.